server/statsPageRedis.js

```diff
@@ -75,6 +75,27 @@ module.exports = function(app, env) {
     try {
       const stats = await getStats()
 
+      const tableOfContents = Object.keys(stats)
+        .map(
+          (subject, i) => `
+          <b>${subject}:</b>
+          ${Object.keys(stats[subject])
+    .map((period, j) =>
+      j % 3 === 0
+        ? `
+            <ul>
+                <a href="#${subject + period}">${
```

Comment:
this introduces colons (":") in the ids which is not good given that it clashes with css selector syntax. I would rather wrap it in a str.replace